jaas/command/src/main/java/org/apache/karaf/jaas/command/ManageRealmCommand.java

```diff
@@ -35,14 +35,14 @@
 @Service
 public class ManageRealmCommand extends JaasCommandSupport {
 
-    @Option(name = "--realm", description = "JAAS Realm", required = false, multiValued = false)
+    @Option(name = "--realm", description = "JAAS Realm. Must be combined with --module", required = false, multiValued = false)
```

Comment:
That's not fully correct. Realm can be combined with module, but also with index.

Reply:
Why should I combine this with index? Index is already a unique attribute.

Reply:
Ah yes, my bad, I changed that. You are right.